pom.xml

```diff
@@ -116,6 +123,46 @@
                             <appName>at-sce-api-qa</appName>
                         </configuration>
                     </plugin>
+                    <!-- JaCoCo -->
```

Comment:
you are missing important exclusions in jacoco's configuration

Reply:
you forgot add classes/packages such as domain, constants, config and exceptions